### PR TITLE
PSP-4066: Add test coverage for document upload, list, get document

### PR DESCRIPTION
**Pull request by Maheshbabu28**

### CODE REVIEW

File: frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        timeStamp: '',
+      },
+      id: 1,
+      document_type: { id: 1, label: 'BC Assessment Search' },
+    },
+    id: 1,
+    metadata_type: { id: 1, label: 'Tag' },
+    url: '',
+    value: 'Tag1234',
+  },
```
Comment (asanchezr) [inline frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx:52]: We are preferring to move these into common **mocks** folder to allow for reuse

File: frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        store: {
+          [lookupCodesSlice.name]: { lookupCodes: mockLookups },
+        },
+        history,
+      },
+    );
+
+    return {
+      ...utils,
+      useMockAuthentication: true,
```
Comment (asanchezr) [inline frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx:80]: I believe `useMockAuthentication: true,` goes in the call to `render(...)` not here...

File: frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    };
+  };
+
+  beforeEach(() => {
+    mockKeycloak({ claims: [Claims.DOCUMENT_VIEW, Claims.DOCUMENT_EDIT] });
```
Comment (asanchezr) [inline frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx:85]: for simplicity you could pass the claims to the render method in the setup function above

File: frontend/src/features/documents/documentUpload/DocumentUploadView.test.tsx [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+      lookup: '',
+      name: 'Tag',
+      parser: '',
+      parser_arguments: '',
+      url: '',
+      validation: '',
+      validation_arguments: '',
+    },
+    required: true,
+  },
```
Comment (asanchezr) [inline frontend/src/features/documents/documentUpload/DocumentUploadView.test.tsx:50]: again, these could be reused if they were moved to shared **mocks** folder

**Review by asanchezr — APPROVED** [review]

File: frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx (hunk 1 above)
Comment (Maheshbabu28) [reply frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx:52]: done

File: frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx (hunk 2 above)
Comment (Maheshbabu28) [reply frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx:80]: done

File: frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx (hunk 3 above)
Comment (Maheshbabu28) [reply frontend/src/features/documents/documentDetail/DocumentDetailView.test.tsx:85]: done

File: frontend/src/features/documents/documentUpload/DocumentUploadView.test.tsx (hunk 4 above)
Comment (Maheshbabu28) [reply frontend/src/features/documents/documentUpload/DocumentUploadView.test.tsx:50]: done

**Pull request merged**